build/pack.ps1

```diff
@@ -84,7 +84,7 @@ function Pack-Image() {
         <# Next, we specify any additional NuGet sources to be used. #> `
         --build-arg EXTRA_NUGET_SOURCES="$extraNugetSources" `
         <# Next, we specify any additional NuGet packages that should be imported. #> `
-        --build-arg EXTRA_NUGET_PACKAGES="" `
+        --build-arg EXTRA_NUGET_PACKAGES=$("microsoft.quantum.providers.ionq", "microsoft.quantum.providers.honeywell", "microsoft.quantum.providers.qci") `
```

Comment:
This package names may be case sensitive? Would also need to check if this gets overwritten in the e2e QDK build.

Reply:
Looking at the internal definition at https://github.com/microsoft/qdk/blob/df63ff5521801cbe16a5fc3b84422f6d9c3308e9/build.vnext/steps-docker.yml#L18, I believe this change may not be effective for the e2e build. I think you've also used PowerShell notation here, but this argument will be processed by Docker as a space-separated list of strings instead?

Reply:
Thanks! I'll make a PR there as well!

For the argument, I did check that it builds and works locally using the same RUN logic and passing in those parameters exactly--but I can swap to space separated list in strings for consistency, if that's how we use it elsewhere!

Reply:
Is there still value in adding it here? Or should I abandon this PR and just go with the internal definition?